platform/logging/LoggingConfig.ts

```diff
@@ -0,0 +1,82 @@
+import * as schema from '../lib/schema';
+import { typeOfSchema } from '../types';
+import { LogLevel } from './LogLevel';
+import { BaseAppenderConfig, BaseAppenderConfigSchemaType } from './appenders/base/BaseAppenderConfig';
+import { ConsoleAppenderConfig } from './appenders/console/ConsoleAppenderConfig';
+import { FileAppenderConfig } from './appenders/file/FileAppenderConfig';
+import { RollingFileAppenderConfig } from './appenders/rolling-file/RollingFileAppenderConfig';
+
+const KNOWN_APPENDERS = new Map<string, new (schema:  BaseAppenderConfigSchemaType) => BaseAppenderConfig>([
```

Comment:
I was thinking about both... but not sure if we ever want plugins to allow adding new loggers.